src/getting-started/installation.md

```diff
@@ -54,6 +56,69 @@ DROP EXTENSION IF EXISTS vectors;
 CREATE EXTENSION vectors;
 ```
 
+## From ZIP package
+
+For systems that are not Debian based and cannot run a Docker container, please follow these steps to install:
+
+1. Before install, make sure that you have the necessary packages installed, including `PostgreSQL`, `pg_config`, `unzip`, `wget`.
+
+```sh
+# Example for RHEL9 dnf
+# Please check your package manager
+sudo dnf install -y unzip wget libpq-devel
+sudo dnf module install -y postgresql:15/server
+sudo postgresql-setup --initdb
+sudo systemctl start postgresql.service
+sudo systemctl enable postgresql.service
+```
+
+2. Verify whether `$pkglibdir` and `$shardir` have been set by `PostgreSQL`. 
+
+```sh
+pg_config --pkglibdir
+# Print something similar to:
+# /usr/lib/postgresql/15/lib or
+# /usr/lib64/pgsql
+
+pg_config --sharedir
+# Print something similar to:
+# /usr/share/postgresql/15 or
+# /usr/share/pgsql
+```
+
+3. Download the zip package in the release page and extract it to a temporary directory.
+
+```sh
+wget https://github.com/tensorchord/pgvecto.rs/releases/download/v0.2.1/vectors-pg15_x86_64-unknown-linux-gnu_0.2.1.zip -O vectors.zip
+unzip vectors.zip -d vectors
+```
+
+4. Copy the extension files to the PostgreSQL directory.
```

Comment:
Users should pack the files to a package for their distros. Just copy the files is not a good idea.

Reply:
Many projects release a dynamic library in the form of a `.so` file, which users must copy to `/usr/lib`.

It's hard for us to instruct user to pack them, as different packer have different usages.